index.html

```diff
@@ -30,56 +30,84 @@
+          </g></svg>
+          </span>
+          <h4 class="main-small-heading">Dogs</h4>
+      </div>
+      <div class="dogs-cats-birds-rabbits-box dogs-cats-box">
```

Comment:
klasa dogs-cats-box jest do wywalenia. Nie robimy takich rzeczy poprzez top: 10%.
Zwróć uwagę na .pets-love-wrapper i zastanów się jak to **ŁADNIE** zrobić.

Reply:
Nie Poprawione